src/status_im2/contexts/shell/share/view.cljs

```diff
@@ -1,21 +1,20 @@
 (ns status-im2.contexts.shell.share.view
-  (:require
-    [clojure.string :as string]
-    [quo.core :as quo]
-    [quo.foundations.colors :as colors]
-    [react-native.blur :as blur]
-    [react-native.core :as rn]
-    [react-native.navigation :as navigation]
-    [react-native.platform :as platform]
-    [reagent.core :as reagent]
-    [status-im.multiaccounts.core :as multiaccounts]
-    [status-im.ui.components.list-selection :as list-selection]
-    [status-im2.common.qr-codes.view :as qr-codes]
-    [status-im2.contexts.shell.share.style :as style]
-    [utils.address :as address]
-    [utils.i18n :as i18n]
-    [utils.image-server :as image-server]
-    [utils.re-frame :as rf]))
+  (:require [clojure.string :as string]
```

Comment:
It would be better IMO if the formatting differences for the `ns` macro are reverted in this PR. I see other files changed too. Maybe your editor is formatting automatically for you like this? I ask this because zprint is not currently configured to keep the first namespace aligned with `:require`, the tool actually respects whatever style we choose, which is a problem because we will keep introducing personal styling choices that only make rebasing more painful since one of the most common conflicts are in the `ns` macro.

Reply:
I did this because we have differences in the formatting style between zprint and clj-kondo (btw, also IntelliJ formats requires as clj-kondo, so I think zprint is not doing it properly). The formatting added is equally styled for everyone.

Anyway, I reverted them in 0eaec990365d8177783a80505d3b63c14ce0df69, since I also don't like this kind of diffs.

Reply:
Yeah, we need to sort this out at once. `zprint` should be driving the formatting, not the other tools, but `zprint` should also be configured a little bit better for the team's taste for namespaces.